### ISSUE TITLE
Alignment of multiple variables in var statement

### ISSUE THREAD

**Issue by goloroden** [eslint/eslint#2068]
At the moment, the following code is considered correct:

```javascript
var x = 2,
  y = 3;
```

It would be awesome if one could specify (maybe using a new rule?) whether normal indentation should be enforced (as with sample above) or whether the variables should be aligned below each other, such as:

```javascript
var x = 2,
    y = 3;
```

**Comment by lo1tuma** [comment]
Looks like a duplicate of #1800.
